cicecore/cicedynB/general/ice_forcing.F90

```diff
@@ -182,6 +182,22 @@ module ice_forcing
          snw_kappa_fname, &  ! snow table 3d kappa field name
          snw_drdt0_fname     ! snow table 3d drdt0 field name
 
+      ! SNICAR snow grain single-scattering properties (SSP) for
+      ! direct (drc) and diffuse (dfs) shortwave incidents
+
+      character (len=char_len_long), public :: &
+         ssp_filename        ! filename for snicar snow and aerosol lookup table
+
+      character (char_len), public :: &
+         ssp_snwgrad_fname,   &  ! snow grain radius field name
+         ssp_specint_fname,   &  ! spectral intervals field name
+         ssp_extcffdrc_fname, &  ! snow mass extinction cross section (m2/kg) field name
+         ssp_extcffdfs_fname, &  ! snow mass extinction cross section (m2/kg) field name
+         ssp_sscalbdrc_fname, &  ! snow single scatter albedo (fraction) field name
+         ssp_sscalbdfs_fname, &  ! snow single scatter albedo (fraction) field name
+         ssp_asmprmdrc_fname, &  ! snow asymmetry factor (cos(theta)) field name
+         ssp_asmprmdfs_fname     ! snow asymmetry factor (cos(theta)) field name
+
```

Comment:
The dEdd code currently uses just df and dr for diffuse and direct -- these names can use that convention too.  How about
ssp_snwradius
ssp_snwspint or ssp_snwspectralbd
ssp_snwextdr(f) or ssp_snwextinctdr(f)
ssp_snwalbdr(f) or ssp_snwalbedodr(f)
ssp_asymmdr(f) or ssp_asymmetrydr(f)
?  Open to discussion.  I generally prefer shorter names that are still long enough to be intuitive.

Reply:
Happy to update these.  Think we want "fname" at the end of these variables as they are the "field name" on the netcdf file, but the data arrays won't have that string.

Reply:
Yes, that's what I intended -- these are the 'stem' names to which suffixes can be added.